app/components/primer/alpha/navigation_list.rb

```diff
@@ -0,0 +1,131 @@
+      #       <% end %>
+      #       <% section.item(selected_by_ids: :password, href: "/account/password") do |item| %>
+      #         Password
+      #         <% item.leading_visual_icon(icon: :key) %>
+      #       <% end %>
+      #       <% section.item(selected_by_ids: :billing, href: "/account/billing") do |item| %>
+      #         Billing info
+      #         <% item.leading_visual_icon(icon: :package) %>
+      #       <% end %>
+      #     <% end %>
+      #   <% end %>
+      #
+      # @example Items with sub-items
+      #
+      #   <%= render(Primer::Alpha::NavigationList.new(aria: { label: "Settings" }, selected_item_id: :email_notifications)) do |component| %>
+      #     <% component.section(aria: { label: "Account settings" }) do |section| %>
+      #       <% section.heading do %>
+      #         Account Settings
+      #       <% end %>
+      #       <% section.item(selected_by_ids: :notifications, label: "Notifications settings") do |item| %>
+      #         Notifications
+      #         <% item.leading_visual_icon(icon: :bell) %>
+      #         <% item.subitem(selected_by_ids: :email_notifications, href: "/account/notifications/email") do |subitem| %>
+      #           Email
+      #           <% subitem.trailing_visual_icon(icon: :mail) %>
+      #         <% end %>
+      #         <% item.subitem(selected_by_ids: :sms_notifications, href: "/account/notifications/sms") do |subitem| %>
+      #           SMS
+      #           <% subitem.trailing_visual_icon(icon: :"device-mobile") %>
+      #         <% end %>
+      #       <% end %>
+      #       <% section.item(selected_by_ids: :messages, label: "Notifications settings") do |item| %>
+      #         Messages
+      #         <% item.leading_visual_icon(icon: :bookmark) %>
+      #         <% item.subitem(href: "/account/messages/inbox") do |subitem| %>
+      #           Inbox
+      #           <% subitem.trailing_visual_counter(count: 10) %>
+      #         <% end %>
+      #         <% item.subitem(href: "/account/messages/organizer") do |subitem| %>
+      #           Organizer
+      #           <% subitem.trailing_visual_label(scheme: :primary) { "New" } %>
+      #         <% end %>
+      #       <% end %>
+      #     <% end %>
+      #   <% end %>
+      #
+      # @param selected_item_id [Symbol] The id of the selected item. Should correspond to one of the item ids in the list.
+      # @param item_classes [Array<String>] Additional classes to add to the list's items.
+      # @param list_arguments [Hash] <%= link_to_system_arguments_docs %>
+      def initialize(selected_item_id: nil, item_classes: "", **list_arguments)
+        @list_arguments = list_arguments
+        @list_arguments[:classes] = class_names(
+          "ActionList",
+          @list_arguments[:classes]
+        )
+
+        aria_label = aria(:label, list_arguments)
+        raise ArgumentError, "an aria-label is required" if aria_label.nil?
```

Comment:
```suggestion
        raise ArgumentError, "an aria-label is required" if aria_label.nil? && !Rails.env.production?
```

Reply:
we could also update

https://github.com/primer/view_components/blob/1b6359e871749c30f62f3d55daed49ea75842fc8/app/components/primer/component.rb#L44-L47

Right now, it looks at `@system_arguments`, but we could pass `@list_arguments` as an argument